README.md

```diff
@@ -45,6 +45,26 @@ If you use GLASS simulations or the GLASS library in your research, please
 [cite the original GLASS paper](https://glass.readthedocs.io/stable/user/publications.html)
 in your publications.
 
+    @ARTICLE{2023OJAp....6E..11T,
+        author = {{Tessore}, Nicolas and {Loureiro}, Arthur and
+        {Joachimi}, Benjamin and {von Wietersheim-Kramsta}, Maximilian
+        and {Jeffrey}, Niall},
+        title = "{GLASS: Generator for Large Scale Structure}",
+        journal = {The Open Journal of Astrophysics},
+        keywords = {Astrophysics - Cosmology and Nongalactic Astrophysics},
+        year = 2023,
+        month = mar,
+        volume = {6},
+        eid = {11},
+        pages = {11},
+        doi = {10.21105/astro.2302.01942},
+        archivePrefix = {arXiv},
+        eprint = {2302.01942},
+        primaryClass = {astro-ph.CO},
+        adsurl = {https://ui.adsabs.harvard.edu/abs/2023OJAp....6E..11T},
+        adsnote = {Provided by the SAO/NASA Astrophysics Data System}
+    }
```

Comment:
Better to use syntax highlighting rather than indenting. GitHub supports `bibtex` highlighting https://github.com/github-linguist/linguist/blob/f164d13fa618023ecf2d8f2ed9a6ce5fae731346/lib/linguist/languages.yml#L579-L590
```suggestion
```bibtex
@article{2023OJAp....6E..11T,
    author = {{Tessore}, Nicolas and {Loureiro}, Arthur and
    {Joachimi}, Benjamin and {von Wietersheim-Kramsta}, Maximilian
    and {Jeffrey}, Niall},
    title = {{GLASS: Generator for Large Scale Structure}},
    journal = {The Open Journal of Astrophysics},
    keywords = {Astrophysics - Cosmology and Nongalactic Astrophysics},
    year = 2023,
    month = mar,
    volume = 6,
    eid = 11,
    pages = 11,
    doi = {10.21105/astro.2302.01942},
    archivePrefix = {arXiv},
    eprint = {2302.01942},
    primaryClass = {astro-ph.CO},
    adsurl = {https://ui.adsabs.harvard.edu/abs/2023OJAp....6E..11T},
    adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}
```
```